slices/git.yaml

```diff
@@ -0,0 +1,27 @@
+package: git
+
+essential:
+  - git_copyright
+
+slices:
+  bins:
+    essential:
+      - libc6_libs
+      - libpcre2-8-0_libs
+      - zlib1g_libs
+    contents:
+      /usr/bin/git:
+      /usr/bin/git-shell:
+      /usr/bin/scalar:
+
+  core:
+    essential:
+      - git_bins
+      - libcurl3t64-gnutls_libs
+      - libexpat1_libs
+    contents:
+      /usr/lib/git-core/**:
```

Comment:
I actually agree with both of you here :)

The thing is that in other SDFs, we use the term `core` to refer to the actual core minimum that is required for the pkg's functionality to work (e.g. `python3.12_core`).

Here, I'm afraid the slice name conveys the opposite, as `bins` is more fundamental than `core`. 

I get that the name is given after the folder name, but what about we compromise and just call it `git-core`?

Reply:
That is a good call! I'll change that.